## optional filter by...
string[] names
string[] types
---
spatial_db_msgs/ObjectInstance[] objects
